src/components/_App/Footer/index.tsx

```diff
@@ -30,6 +30,19 @@ const StyledFooter = styled.footer`
         opacity: 1;
       }
     }
+    &.pages {
+      display: block;
+      columns: 4 auto;
```

Comment:
add mobile version. Auto doesn't work properly

```suggestion
      columns: ${isMobile ? "1 auto" : "4 auto"};
      text-align: ${isMobile ? "center" : "left"};
```

![image](https://user-images.githubusercontent.com/32429974/174845878-975cb1c1-e8cf-4798-bb09-538db88c2e0d.png)